tooling/bundle-content-scripts.ts

```diff
@@ -45,6 +45,7 @@ buildContentScript(([] as string[]).concat(
   getFilesInDir(`${sourceDir}/common/browser`, /\.js$/, false),
   getFilesInDir(`${sourceDir}/common`, /\.js$/, false),
   getFilesInDir(`${sourceDir}/content_scripts/webmail`, /\.js$/),
+  './extension/lib/sweetalert2.js',
```

Comment:
I think this can be moved to manifest.json instead (as written here, the lib would be packaged twice - once as `   'extension/lib/sweetalert2.js` and once bundled into content script). Since it's a self-contained library, we may be able to include it in manifest directly. I'll try.